### PR TITLE
Operator plaintext key field removal

### PR DESCRIPTION
**Pull request by gmsdelmundo**

### CODE REVIEW

**Review by vishalegbert-ttd — APPROVED** [review]

File: webroot/adm/client-key.html [hunk 1]
```diff
@@ -82,17 +82,6 @@ <h3>Output</h3>
             doApiCall('POST', url, '#standardOutput', '#errorOutput');
         });
 
-        $('#doDel').on('click', function () {
```
Comment (vishalegbert-ttd) [inline webroot/adm/client-key.html:85]: Why?

File: src/main/java/com/uid2/admin/vertx/service/ClientJsonParser.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import com.uid2.shared.util.Mapper;
+import io.vertx.core.json.JsonObject;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collection;
+
+// UID2-1503: Temporary for backwards compatibility
+public class ClientJsonParser implements Parser<Collection<JsonObject>> {
```
Comment (vishalegbert-ttd) [inline src/main/java/com/uid2/admin/vertx/service/ClientJsonParser.java:16]: Still needed?

File: webroot/adm/client-key.html (hunk 1 above)
Comment (gmsdelmundo) [reply webroot/adm/client-key.html:85]: We don't support client delete anymore, should we still keep it?

File: src/main/java/com/uid2/admin/vertx/service/ClientJsonParser.java (hunk 2 above)
Comment (gmsdelmundo) [reply src/main/java/com/uid2/admin/vertx/service/ClientJsonParser.java:16]: Nope, deleted

**Pull request merged**